pipeline/stop_places/sql/stop_places.sql

```diff
@@ -101,20 +101,96 @@ LANGUAGE SQL IMMUTABLE STRICT;
 
 
 /*
- * Create a keyList element based on a delfi attribut to osm matching
+ * Create a single key value pair element where the value is empty if the given tag value equals to the given value
+ * Else returns null
+ */
+CREATE OR REPLACE FUNCTION delfi_attribute_check_value_xml(delfiid text, val text, val2 text) RETURNS xml AS
+$$
+SELECT CASE
+  WHEN $2 = $3 THEN create_KeyValue($1, '')
+END
+$$
+LANGUAGE SQL IMMUTABLE STRICT;
+
+
+/*
+ * Create a single key value pair element where the value is empty if the given tag value equals to one of the given array values
+ * Else returns null
+ */
+CREATE OR REPLACE FUNCTION delfi_attribute_check_values_array_xml(delfiid text, val text, valArray text[]) RETURNS xml AS
+$$
+BEGIN
+  IF val = ANY (valArray) THEN
+    RETURN create_KeyValue($1, '');
+  END IF;
+  RETURN NULL;
+END
+$$
+LANGUAGE plpgsql IMMUTABLE STRICT;
```

Comment:
I like the `delfi_attribute_check_values_array_xml` function and would like to combine the following functions to one.

- `delfi_attribute_on_yes_xml`
- `delfi_attribute_check_value_xml`
- `delfi_attribute_check_values_array_xml`

Also I would prefer using the `VARIADIC` keyword to be able to pass a variable number of arguments without having to construct an array:
```SQL
CREATE OR REPLACE FUNCTION delfi_attribute_check_values_xml(delfiid text, val text, VARIADIC values text[]) RETURNS xml AS
$$
BEGIN
  IF val = ANY (values) THEN
    RETURN create_KeyValue($1, '');
  END IF;
  RETURN NULL;
END
$$
LANGUAGE plpgsql IMMUTABLE STRICT;
```

If possible you could also set "yes" as the default value for the `values` parameter. Perhaps via `VARIADIC values text[] DEFAULT ['yes']`

Reply:
I now have combined all of those functions into one [8a158dc](https://github.com/OPENER-next/osm2vdv462/pull/27/commits/8a158dc66d3556c5363d1e3e5fe969311ea6f145)